### ISSUE TITLE
BatchNorm/beta not found error

### ISSUE THREAD

**Issue by Naykira** [wbenbihi/hourglasstensorlfow#45]
Hello, thanks a lot for the tensorflow implementation of hourglass architecture. I have been trying to test the models and I am running into the below error during inference;

Tensor name "BatchNorm/beta/RMSProp" not found in checkpoint files /mnt//hourglass/hourglass/RegHG/hg_refined_200.index

Has anyone else encountered this issue? I would be grateful for any suggestions regarding how this could be solved. Thanks!

**Comment by techjun** [comment]
When I used hg_refined_200_tiny occurred same error.

but hg_refined_200 works well.

**Comment by Naykira** [comment]
Would it be possible to please share the changes, if any, you made to the code to get the  hg_refined_200 working? Because I am facing the above issue with both tiny and normal hg implementation!

**Comment by techjun** [comment]
https://github.com/wbenbihi/hourglasstensorlfow/blob/356e0638a1bf28ecc92a121812e74d3fd0e353f5/predictClass.py#L1280-L1287

I just changed it to config.cfg   and predict.load_model(load = 'hg_refined_200')

**Comment by Naykira** [comment]
I can get it working now but it's leading to wrong detection of body keypoints. Maybe it's because the original images are resized to accommodate the network. But thanks a lot! Do you any idea why the batchnorm error comes up and how it can probably be solved?
